packages/server/utils/analytics/amplitude/AmplitudeAnalytics.ts

```diff
@@ -0,0 +1,29 @@
+import {init, track} from '@amplitude/analytics-node'
+import {AnalyticsEvent} from '../analytics'
+import {getUserById} from '../../../postgres/queries/getUsersByIds'
+import PROD from '../../../PROD'
+
+const {AMPLITUDE_WRITE_KEY} = process.env
+
+/**
+ * Wrapper for amplitude providing a more typesafe interface
+ */
+export class AmplitudeAnalytics {
+  constructor() {
+    init(AMPLITUDE_WRITE_KEY || 'x', {
+      flushQueueSize: PROD ? 20 : 1,
+      optOut: !AMPLITUDE_WRITE_KEY
+    })
+  }
+
+  async track(userId: string, event: AnalyticsEvent, properties?: Record<string, any>) {
+    // used as a failsafe for PPMIs
+    if (!AMPLITUDE_WRITE_KEY) return
+    const user = await getUserById(userId)
+    const {email} = user ?? {}
```

Comment:
Is `email` required? Reading from this [section](https://www.docs.developers.amplitude.com/data/sdks/typescript-node/#tracking-an-event), it looks like only `user_id` is needed.

Reply:
`email` is not required, but we are currently adding email to each segment event we emit server side, so I used the same logic to ensure parity between the two implementations. 
 https://github.com/ParabolInc/parabol/blob/699c1d423796befdd3b1dda9360a1c3a14f2f254/packages/server/utils/segmentIo.ts#L17-L19